SDK Showcase/SDK Showcase/Views/ContentView.swift

```diff
@@ -138,19 +150,21 @@ extension ContentView {
     }
     
     func buttonAction(for selection: Selection) {
-        switch selection.type {
+        let switcher = selection.type == .unknown ? selection.namedType : selection.type
```

Comment:
What is it for? You switch by type anyway. BTW the only I guess advantage of `switcher` is not used anyway because  at the end of switch statement you check for `.unknown` the you use `selection.name` instead of `switcher` that would be a `selection.name` assigned here. What have I missed?